services/notifications/buildspec.yml

```diff
@@ -0,0 +1,24 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo Logging in to Amazon ECR...
+      - aws --version
+      - $(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)
+      - REPOSITORY_URI=461497099412.dkr.ecr.us-east-1.amazonaws.com/hackillinois-api-notifications
```

Comment:
This is correct. I'll need to actually create this repo though.

(This is just a reminder for me)